fuzz/client.c

```diff
@@ -25,6 +26,23 @@ extern int rand_predictable;
 /* unused, to avoid warning. */
 static int idx;
 
+#define FUZZTIME 1485898104
+
+#ifdef _WIN32
+# ifdef _TIME64_T_DEFINED
+time64_t _time64(time64_t *t)
+# else
+time_t _time(time_t *t)
```

Comment:
Do note that `time` implementation would be *omitted* on VC compiler. I.e. in VC build you are likely to get _time64 implementation alone, while in say mingw build you'll get _time64 and time, And of course in all other cases you'll get time alone. [One can argue in favour of squeezing in _time32, but can't see that it's that common.] [Off for today...]